java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -132,4 +133,9 @@ public CompletableFuture<String[]> time() {
         return commandManager.submitNewCommand(
                 Time, new String[0], response -> castArray(handleArrayResponse(response), String.class));
     }
+
+    @Override
+    public CompletableFuture<Long> lastsave() {
```

Comment:
Not sure if this should be camel-case or left all lower-case?
```suggestion
    public CompletableFuture<Long> lastSave() {
```

Reply:
All lowercase, except two-word commands like `clientId`, etc